src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1585,6 +1585,8 @@ file.metaData.checksum.copy=Click to copy
 file.metaData.dataFile.dataTab.unf=UNF
 file.metaData.dataFile.dataTab.variables=Variables
 file.metaData.dataFile.dataTab.observations=Observations
+file.metaData.access.restricted=File Access: Restricted
+file.metaData.access.granted=File Access: Restricted with Access Granted
```

Comment:
Minor, but it feels these could be broken up - one bundle for "File Access" and another (two) for "Restricted" and "Restricted with Access Granted". The reason is because further down in the bundle "Restricted" and "Restricted with Access Granted" and defined again for the download button. So if a change is made, it would have to be on both places (assuming that the same text is wanted in both). And I'm sure there are other places we use "Restricted" and "Restricted with Access Granted". All that said, that could maybe be a cleanup for some other time, but wanted to mention here in case we wanted to get a head start.

Reply:
For example the bundle also has:
restricted=Restricted
restrictedaccess=Restricted Access

(not sure if the 2nd one is the equivalent of "Restricted with Access Granted" but feels like it would be)

Reply:
Fixed in 6f8ddc933. I added "Restricted with Access Granted" as its own entry because I don't know its relationship to "Restricted Access".

Reply:
I added a new note for this (after a quick check of where "Restricted Access" was used. (I should've probably just replied here - sorry.